<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sysMonitor</class>
 <widget class="QWidget" name="sysMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System monitor</string>
  </property>
  <widget class="QTabWidget" name="selectTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>576</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab{width:90}
QTabBar::tab{height:30}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Process</string>
    </attribute>
    <widget class="QTableWidget" name="processInfo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-6</y>
       <width>720</width>
       <height>511</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView::item { border-left: 0px solid black; }</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>PID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PPID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Threads</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Memory</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="killProcessBtn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>510</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Kill</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>CPU</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>640</width>
       <height>310</height>
      </rect>
     </property>
     <property name="title">
      <string>CPU Usage</string>
     </property>
     <widget class="QChartView" name="cpuUsageChart" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>580</width>
        <height>220</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Usage:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpuUsageLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>640</width>
       <height>200</height>
      </rect>
     </property>
     <property name="title">
      <string>CPU Information</string>
     </property>
     <widget class="QLabel" name="cpuRatioTest">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cpuInfoLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>480</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>480</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Type and Frequency:</string>
      </property>
     </widget>
     <widget class="QLabel" name="coreNumLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>480</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>480</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU cores:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Memory</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>640</width>
       <height>310</height>
      </rect>
     </property>
     <property name="title">
      <string>Memory Usage</string>
     </property>
     <widget class="QChartView" name="memoryUsageChart" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>580</width>
        <height>220</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory Usage:</string>
      </property>
     </widget>
     <widget class="QLabel" name="memoryUsageLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>System</string>
    </attribute>
    <widget class="QGroupBox" name="formGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>501</height>
      </rect>
     </property>
     <property name="title">
      <string>System Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="localHostNameHint">
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>电脑名称：   </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="localHostNameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="osInfoHint">
        <property name="text">
         <string>操作系统:   </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="osInfoLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="osTypeHint">
        <property name="text">
         <string>OS Type:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="osTypeLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>磁盘信息：</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="diskInfoLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>About</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
